android/PhysicalWeb/app/src/main/res/layout/list_item_blocked.xml

```diff
@@ -0,0 +1,33 @@
+<?xml version="1.0" encoding="utf-8"?>
+
+<RelativeLayout xmlns:android="http://schemas.android.com/apk/res/android"
+              android:orientation="horizontal"
+              android:paddingStart="18dp"
+              android:paddingEnd="18dp"
+              android:paddingTop="18dp"
+              android:paddingBottom="18dp"
+              android:layout_width="match_parent"
+              android:layout_height="wrap_content"
+              android:background="#FFFFFF">
+
+    <TextView
+            android:id="@+id/blocked_host"
+            android:layout_width="wrap_content"
+            android:layout_height="wrap_content"
+            android:textSize="15sp"
+            android:layout_toStartOf="@+id/delete_button"
+            android:fontFamily="arial-sans-serif"
+            android:textColor="@color/description_color"
+            android:layout_alignParentStart="true">
```

Comment:
in addition to layout_alignParentStart="true", also add:

android:layout_alignParentTop="true"
android:layout_toStartOf="@id/delete_button"